app/controllers/sessions_controller.rb

```diff
@@ -54,12 +54,12 @@ def team_member?(*args)
   end
 
   def member?(client, nickname, type:)
-    id = ENV["GITHUB_#{type.to_s.upcase}_ID"]
+    id = Octobox.config.public_send("github_#{type}_id")
```

Comment:
I'm not a big fan of using public_send when a case or if statement will work, I would suggest changing it, but there's another a few lines down so we may as well be consistent.